apps/keyboard/js/layouts/th.js

```diff
@@ -0,0 +1,134 @@
+Keyboards.th = {
+  label: 'Thai',
+  shortLabel: 'Th',
+  menuLabel: 'ไทย', /*ไทย*/
+  secondLayout: true,
+  basicLayoutKey: 'กขค', /*กขค*/
+  imEngine: 'latin',
+  types: ['text', 'url', 'email'],
+  width: 11,
+  keys: [
```

Comment:
we already change the format of the layout definition to allow it contain several pages, see the following file as an example.
https://github.com/mozilla-b2g/gaia/blob/master/apps/keyboard/js/layouts/gl.js#L8